content/blog/gain-platform-superpowers-with-the-kebap-stack/index.md

```diff
@@ -0,0 +1,704 @@
+                            parent resource option specifies a parent for a resource. It is used to associate
+                            children with the parents that encapsulate or are responsible for them.
+                          x-kubernetes-preserve-unknown-fields: true
+                        protect:
+                          description: protect prevents accidental deletion of a resource.
+                          type: boolean
+                        provider:
+                          description: provider resource option sets a provider for
+                            the resource.
+                          x-kubernetes-preserve-unknown-fields: true
+                        providers:
+                          additionalProperties:
+                            x-kubernetes-preserve-unknown-fields: true
+                          description: providers resource option sets a map of providers
+                            for the resource and its children.
+                          type: object
+                        version:
+                          description: version specifies a provider plugin version
+                            that should be used when operating on a resource.
+                          type: string
+                      type: object
+                    properties:
+                      additionalProperties:
+                        x-kubernetes-preserve-unknown-fields: true
+                      description: properties contains the primary resource-specific
+                        keys and values to initialize the resource state.
+                      type: object
+                    type:
+                      description: type is the Pulumi type token for this resource.
+                      minLength: 1
+                      type: string
+                  required:
+                  - type
+                  type: object
+                description: resources declares the Pulumi resources that will be
+                  deployed and managed by the program.
+                type: object
+              variables:
+                additionalProperties:
+                  x-kubernetes-preserve-unknown-fields: true
+                description: |-
+                  variables specifies intermediate values of the program; the values of variables are
+                  expressions that can be re-used.
+                type: object
+            type: object
+        type: object
+    served: true
+    storage: true
+    subresources: {}
+```
+
+### Argo CD
+
+<img src="a.png">
+
+[Argo CD](https://argo-cd.readthedocs.io/en/stable/) handles the deployment layer, ensuring that the current deployment
+matches what is declaratively defined in the
+source. It's emerging as the de facto standard due to its rich front-end and powerful features. Argo CD implements the
+GitOps model, continuously monitoring your Git repositories and automatically updating the deployed applications to
```

Comment:
I think we need to discuss whether this is deploying data plane stuff or deploying the workloads to run in the K8s cluster or is it deploying the Pulumi infrastructure as code programs? What is actually being deployed?